### PR TITLE
Create db

### PR DESCRIPTION
**Pull request by alaabadra**

### CODE REVIEW

File: server/database/connection.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+const { Pool } = require('pg');
+const url = require('url');
+require('dotenv').config();
+
+const DB_URL = process.env.DATABASE_URL;
+const allInfoConn = url.parse(DB_URL);
+const [user, password] = allInfoConn.auth.split(':');
+const someInfoConn = {
```
Comment (FarahZaqout) [inline server/database/connection.js]: this is better named ```connection options```

File: server/database/connection.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+const { Pool } = require('pg');
+const url = require('url');
+require('dotenv').config();
+
+const DB_URL = process.env.DATABASE_URL;
```
Comment (FarahZaqout) [inline server/database/connection.js]: use camelcase in javascript

File: server/database/connection.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+const allInfoConn = url.parse(DB_URL);
```
[the remark this answers is not in the document]
Comment (FarahZaqout) [reply server/database/connection.js]: ```allInfoConn``` is not a good name.

File: server/database/build.sql [hunk 4]
```diff
@@ -0,0 +1,23 @@
+BEGIN;
+DROP TABLE IF EXISTS users, tickets_status, tickets CASCADE;
+CREATE TABLE users(
+    id SERIAL PRIMARY KEY,
+    email TEXT NOT NULL,
+    name VARCHAR NOT NULL UNIQUE,
```
Comment (FarahZaqout) [inline server/database/build.sql]: the unique value should be the email, not the name. What if two people are named mohammed?

Comment (alaabadra) [reply server/database/build.sql]: Ohhhh , ok 
But I wrote unique in name instead email .
thank you for this comment

**Pull request closed without merging**